src/store/embalm/actions.ts

```diff
@@ -374,11 +354,4 @@ export function disableSteps(): EmbalmActions {
   };
 }
 
-export function enableSteps(): EmbalmActions {
-  return {
-    type: ActionType.EnableSteps,
-    payload: {},
-  };
-}
```

Comment:
I think this should stay -- the code using this function is temporarily commented out, but we will need re-enable the steps here (unless we change the "disable steps" function to "toggle steps")